queue/util/generateDeadLetterExchangeName.js

```diff
@@ -0,0 +1,6 @@
+/**
+* Generates a dead letter exchange name from a queueName
+*/
+module.exports = function (queueName) {
+  return `${queueName}-dead-letter-exchange`
+}
```

Comment:
This is minor, but I think you should add a unit test for this file asserting that it always starts with `queueName` and ends with `-dead-letter-exchange`, so nobody decides to change this.